tests/PathTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace OpenCFP\Test;
+
+use OpenCFP\Environment;
+use OpenCFP\Path;
+
+/**
+ * @covers \OpenCFP\Path
+ */
+class PathTest extends BaseTestCase
+{
+    /**
+     * @var Path
+     */
+    private $path;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->path = new Path('/home/folder/base', Environment::testing());
+    }
+
+    /**
+     * @test
```

Comment:
Not sure what the consensus is, but as of the moment it seems we have three different variations of test methods

* prefixed with `test` and using `camelCase`
* annotated with `@test` and using `camelCase`
* annotated with `@test` and using `snake_case`

Maybe we should make it consistent across the code base?

What do you think, @chartjes and @mdwheele (since you are the maintainers)?